content/hardware/02.hero/boards/uno-r4-wifi/tutorials/cheat-sheet/cheat-sheet.md

```diff
@@ -398,6 +398,8 @@ Wire.endTransmission(); //stop transmit
 
 ![Qwiic Connector on UNO WiFi R4](assets/QWIIC.png)
 
+***The Qwiic connector on the UNO R4 WiFi is connected to the secondary I2C bus, meaning that it uses the Wire1 object rather than the Wire object.***
```

Comment:
```suggestion
***The Qwiic connector on the UNO R4 WiFi is connected to the secondary I2C bus (IIC0), which uses the `Wire1` object rather than the `Wire` object. Please note that the Qwiic connector is 3.3 V only.***
```